charts/eks-anywhere-packages/templates/credentialpackage.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.cronjob.suspend }}
```

Comment:
What are the cases for this to be false?

Reply:
They want to use the cronjob instead of credential package (for example not wanting to give node level access to registries)

Reply:
We should make that configuration name more obvious then. `UseKubeletCredentials` / `UseImagePullSecret`

Reply:
Will move this to seperate issue as this would require a CLI change to be able to activate the previous cronjob